pgtopoeditor.py

```diff
@@ -19,20 +19,20 @@
  ***************************************************************************/
 """
 # Import the PyQt and QGIS libraries
-from PyQt4.QtCore import *
-from PyQt4.QtGui import *
-from qgis.core import *
+from qgis.PyQt.QtGui import QIcon
+from qgis.PyQt.QtWidgets import QAction, QMessageBox, QProgressBar
+from qgis.core import Qgis, QgsDataSourceUri
 from qgis.gui import QgsMessageBar
 # Initialize Qt resources from file resources.py
-import resources
+from . import resources
 # Import the code for the dialog
-from pgtopoeditordialog import PgTopoEditorDialog
+from .pgtopoeditordialog import PgTopoEditorDialog
 
 import psycopg2
 
 def getIntAttributeByIndex(feature, index):
   # QMessageBox.information(None, '?', "Version of qgis is " + str(QGis.QGIS_VERSION_INT))
-  if QGis.QGIS_VERSION_INT < 10900:
+  if Qgis.QGIS_VERSION_INT < 10900:
```

Comment:
If the API changed, can we make an alias/typedef something to keep the same code for both old and new API ?

Reply:
This version should probably be bumped. I think keeping the same code for both version will be messy and hard to develop further on.

Reply:
since we don't support QGIS < 3.0 anyway, this whole block of code can be simplified